internal/xds/translator/translator_test.go

```diff
@@ -11,14 +11,15 @@ import (
 	"path/filepath"
 	"testing"
 
+	"github.com/envoyproxy/gateway/internal/ir"
 	"github.com/envoyproxy/go-control-plane/pkg/cache/types"
 	resource "github.com/envoyproxy/go-control-plane/pkg/resource/v3"
+	ratelimitserviceconfig "github.com/envoyproxy/ratelimit/src/config"
 	"github.com/stretchr/testify/require"
 	"google.golang.org/protobuf/encoding/protojson"
 	"google.golang.org/protobuf/proto"
+	goyaml "gopkg.in/yaml.v3"
```

Comment:
use `sigs.k8s.io/yaml` instead

Reply:
tried, doesnt work with it, the tags are `yaml` tags, which is why I add to add this package

Reply:
merge this first, I will review depguard settings.